Sources/AutorestSwift/Models/OperationGroup.swift

```diff
@@ -13,6 +13,22 @@ public struct OperationGroup: Codable {
 
     public let operations: [Operation]
 
+    enum CodingKeys: String, CodingKey {
+        case operations, key = "$key"
+    }
+    
     // TODO: Apply allOf
     // public let allOf: [Metadata]?
+    
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        try container.encode(key, forKey: .key)
+        try container.encode(operations, forKey: .operations)
+    }
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        key = try container.decode(String.self, forKey: .key)
+        operations = try container.decode([Operation].self, forKey: .operations)
+    }
```

Comment:
Does the compiler require these? I would think that since we are providing a custom `CodingKeys` enum it should still be able to synthesize these.

Reply:
Thanks. I test it. Yes, the compile does not require these encode/decode functions. I will remove them.